### PR TITLE
Fix UI based on user feedback

### PR DESCRIPTION
**Pull request by aguycalled**
Partially solves the list of issues from user feedback.

https://docs.google.com/document/d/16XNsQQypcKlOSKMF4bcNK2M6JnBNyW1_Rg71q0So7s0/edit#

I've updated the document indicating those which are solved in this pull request

### CODE REVIEW

File: screens/wallet/SettingsScreen.tsx [hunk 1]
```diff
@@ -131,7 +131,7 @@ const SettingsScreen = (props: ScreenProps<'SettingsScreen'>) => {
       onPress: () => biometricsAlert(),
     },
     {
-      title: 'Setup Staking Nodes',
+      title: 'Staking Nodes',
```
Comment (tsejerome) [inline screens/wallet/SettingsScreen.tsx:134]: Why are we not making Nodes lowercase too?

File: screens/wallet/SettingsScreen.tsx [hunk 2]
```diff
@@ -141,7 +141,7 @@ const SettingsScreen = (props: ScreenProps<'SettingsScreen'>) => {
       },
     },
     {
-      title: 'Setup Electrum Servers',
+      title: 'Electrum Servers',
```
Comment (tsejerome) [inline screens/wallet/SettingsScreen.tsx:144]: shd we have servers as lowercase?

File: screens/wallet/SettingsScreen.tsx (hunk 1 above)
Comment (aguycalled) [reply screens/wallet/SettingsScreen.tsx:134]: yes

File: screens/wallet/SettingsScreen.tsx (hunk 2 above)
Comment (aguycalled) [reply screens/wallet/SettingsScreen.tsx:144]: yes

**Review by tsejerome — APPROVED** [review]

**Pull request merged**